src/lib/serviceRoutinesV2/putEntityAttributeValue.cpp

```diff
@@ -79,6 +81,15 @@ std::string putEntityAttributeValue
     }
   }
 
+  if (ciP->httpStatusCode == SccConflict)
+  {
+    ErrorCode   ec("TooManyResults", "There is more than one entity that match the update. Please refine your query.");
```

Comment:
I like

"There is more than one entity with that id. Refine your query."

In addition, to avoid having "diperssion" in text messages, maybe it makes sense to have a #define for this (not sure in which .h file... maybe we could start a "textMessages.h" file in the common library.